### ISSUE TITLE
Live Reload code in web worker

### ISSUE THREAD

**Issue by srobertson421** [darionco/rollup-plugin-web-worker-loader#25]
When I am running svelte in dev mode, some livereload code is being added to the Base64 web worker code, even though the setting says to skip livereload plugin:

rollup.config.js
```
import svelte from 'rollup-plugin-svelte';
import resolve from '@rollup/plugin-node-resolve';
import commonjs from '@rollup/plugin-commonjs';
import livereload from 'rollup-plugin-livereload';
import { terser } from 'rollup-plugin-terser';
import webWorkerLoader from 'rollup-plugin-web-worker-loader';

const production = !process.env.ROLLUP_WATCH;

export default {
	input: 'src/main.js',
	output: {
		sourcemap: true,
		format: 'iife',
		name: 'app',
		file: 'public/build/bundle.js'
	},
	plugins: [
		svelte({
			// enable run-time checks when not in production
			dev: !production,
			// we'll extract any component CSS out into
			// a separate file - better for performance
			css: css => {
				css.write('public/build/bundle.css');
			}
		}),

		// If you have external dependencies installed from
		// npm, you'll most likely need these plugins. In
		// some cases you'll need additional configuration -
		// consult the documentation for details:
		// https://github.com/rollup/plugins/tree/master/packages/commonjs
		resolve({
			browser: true,
			dedupe: ['svelte']
		}),
		commonjs(),

		// In dev mode, call `npm run start` once
		// the bundle has been generated
		!production && serve(),

		// Watch the `public` directory and refresh the
		// browser on changes when not in production
		!production && livereload('public'),

		// If we're building for production (npm run build
		// instead of npm run dev), minify
    production && terser(),
    
    webWorkerLoader({
      targetPlatform: 'browser',
      extensions: ['.worker.js'],
      skipPlugins: [ 'liveServer', 'serve', 'livereload', 'terser' ]
    })
	],
	watch: {
		clearScreen: false
	}
};

function serve() {
	let started = false;

	return {
		writeBundle() {
			if (!started) {
				started = true;

				require('child_process').spawn('npm', ['run', 'start', '--', '--dev'], {
					stdio: ['ignore', 'inherit', 'inherit'],
					shell: true
				});
			}
		}
	};
}

```

testWorker.worker.js
```
onmessage = function(e) {
  console.log('message in worker');
}
```

main.js
```
import App from './App.svelte';
import TestWorker from 'web-worker:./testWorker.worker.js';

const testWorker = new TestWorker();
testWorker.postMessage('tester');

const app = new App({
	target: document.body
});

export default app;
```

Base64 worker
```
(function(l, r) { if (l.getElementById('livereloadscript')) return; r = l.createElement('script'); r.async = 1; r.src = '//' + (window.location.host || 'localhost').split(':')[0] + ':35729/livereload.js?snipver=1'; r.id = 'livereloadscript'; l.getElementsByTagName('head')[0].appendChild(r) })(window.document);
onmessage = function(e) {
  console.log('message in worker');
};
```

browser console error: `Uncaught ReferenceError: window is not defined`

**Comment by darionco** [comment]
Interesting, I'll check this one out and get back to you. Thanks for the report.

**Comment by darionco** [comment]
Fixed in 1.3.1

**Comment by srobertson421** [comment]
Thanks! 👍
